client/src/components/ClassCard.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Card, Button } from 'semantic-ui-react';
+
+import { editUserClasses } from '../utils/apiWrapper';
+import '../css/ClassCard.scss';
+
+
+function ClassCard(props) {
+  const { classCardText } = props;
+  return (
+    <Card>
```

Comment:
Make the styling changes I suggested for a cleaner look:
- shrink the X button and move it closer to the right-hand-side
- enlarge the classCardText
- center the classCardText in the card

![ClassCardBasicSketch](https://user-images.githubusercontent.com/34987983/142182329-e29b8249-ee1b-4bdb-84e6-e6c4511df2ab.jpg)